inst/utils-0.0.1/addCountrySelect.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Adds a dropdown to highlight the selected country in the chart
+ *
+ * @param {Node} el - an element in the DOM
+ * @param {Array} results - KRI results (dfSummary)
+ * @param {Object} instance - Chart.js chart object
+ *
+ * @returns {Node} HTML select element
+ */
+
+
+// Add event listener to highlight countries.
+const addCountrySelect = function(el, results, instance, groupSelect) {
+
+    el.style.position = 'relative';
+
+    // add container in which to place dropdown
+    const countrySelectContainer = document.createElement('div');
+    el.appendChild(countrySelectContainer);
+
+    // position container absolutely in upper left corner of `el`.
+    countrySelectContainer.style.position = 'absolute';
+    countrySelectContainer.style.top = 0;
+    countrySelectContainer.style.left = "175px";
```

Comment:
This looks perfect with three-digit IDs but will overlap with longer IDs.  A `div` that contains both the site select and country select is a more robust approach, something like:

```
<div style = 'position:absolute; top:0'>
    <div class = 'chart-select site-select' style = 'display:inline-block'>...</div>
    <div class = 'chart-select country-select' style = 'display:inline-block'>...</div>
</div>
```

Reply:
Hey @samussiah, I made the changes but we'll see the filters overlap with the canvas. In order to work around this problem, we'd have to make changes to `rbm-viz`, I believe. Should we go ahead and log a ticket for that? 

![image](https://github.com/Gilead-BioStats/gsm/assets/76186904/3b05b1db-7212-485b-aa61-8ba977e34abe)

Reply:
This can likely be addressed with inline CSS or even a stylesheet specifically for these dropdowns.